monitoring/grafana/dashboards/jsonnet/grafana_dashboards.jsonnet

```
@@ -0,0 +1,42 @@
+local grafana = import '../../../../grafonnet-lib/grafonnet/grafana.libsonnet';
+local dashboard = grafana.dashboard;
+local template = grafana.template;
+local graphPanel = grafana.graphPanel;
+local prometheus = grafana.prometheus;
+local annotation = grafana.annotation;
+local tablePanel = grafana.tablePanel;
+local dashboardSchema(title, uid, time_from, refresh, schemaVersion, tags,timezone, timepicker) =
+dashboard.new(title=title,uid=uid,time_from=time_from,refresh=refresh,schemaVersion=schemaVersion,tags=tags,timezone=timezone,timepicker=timepicker);
+local graphPanelSchema(title, nullPointMode, stack, formatY1, formatY2, labelY1, labelY2, min, fill, datasource) =
+graphPanel.new(title=title,nullPointMode=nullPointMode,stack=stack,formatY1=formatY1,formatY2=formatY2,labelY1=labelY1,labelY2=labelY2,min=min,fill=fill,datasource=datasource);
+local addTargetSchema(expr, intervalFactor, format, legendFormat) =
+prometheus.target(expr=expr,intervalFactor=intervalFactor,format=format,legendFormat=legendFormat);
+local addTemplateSchema(name, datasource, query, refresh, hide, includeAll, sort) =
+template.new(name=name,datasource=datasource,query=query,refresh=refresh,hide=hide,includeAll=includeAll, sort=sort);
+local addAnnotationSchema(builtIn, datasource, enable, hide, iconColor, name, type) =
+grafana.annotation.datasource(builtIn=builtIn,datasource=datasource,enable=enable,hide=hide,iconColor=iconColor,name=name,type=type);
+{
+  "radosgw-sync-overview.json":
+    local RgwSyncOverviewPanel(title, formatY1, labelY1, rgwMetric, x, y, w, h) =
+    graphPanelSchema(title, 'null as zero', true, formatY1, 'short', labelY1, null, 0, 1, '$datasource')
+    .addTargets([addTargetSchema('sum by (source_zone) (rate(%s[30s]))' % rgwMetric, 1, 'time_series', '{{source_zone}}')]) + {gridPos: {x: x, y: y, w: w, h: h}};
+
+    dashboardSchema('RGW Sync Overview', 'rgw-sync-overview', 'now-1h', '15s', 16, ["overview"], '', {refresh_intervals:['5s','10s','15s','30s','1m','5m','15m','30m','1h','2h','1d'],time_options:['5m','15m','1h','6h','12h','24h','2d','7d','30d']})
+
+    .addAnnotation(addAnnotationSchema(1,'-- Grafana --',true,true,'rgba(0, 211, 255, 1)','Annotations & Alerts','dashboard'))
+
+    .addRequired(type= 'grafana',id= 'grafana',name= 'Grafana',version= '5.0.0')
+
+    .addRequired(type= 'panel',id= 'graph',name= 'Graph',version= '5.0.0')
```

Are you sure the linter/formatter is ok with this. Usually whitespaces happen after commas, not after = or other binary operators. I see this style issue happening a lot, and maybe it's just me, but it makes it hard for me to read the jsonnet code. 

```suggestion
    .addRequired(type='panel', id='graph', name='Graph', version='5.0.0')
```

Can you please, if not too complex, add a [linter for jsonnet](https://jsonnet.org/learning/tools.html)?